pkg/fftypes/operation.go

```diff
@@ -26,8 +26,8 @@ var (
 	OpTypeBlockchainBatchPin OpType = ffEnum("optype", "blockchain_batch_pin")
 	// OpTypeBlockchainInvoke is a smart contract invoke
 	OpTypeBlockchainInvoke OpType = ffEnum("optype", "blockchain_invoke")
-	// OpTypePublicStorageBatchBroadcast is a public storage operation to store broadcast data
-	OpTypePublicStorageBatchBroadcast OpType = ffEnum("optype", "publicstorage_batch_broadcast")
+	// OpTypeSharedStorageBatchBroadcast is a shared storage operation to store broadcast data
+	OpTypeSharedStorageBatchBroadcast OpType = ffEnum("optype", "sharedstorage_batch_broadcast")
```

Comment:
@peterbroadhurst @awrichar I do question whether renaming this will cause problems with existing deployments that already have operations of type `publicstorage_batch_broadcast` in the DB. Should we do something differently here?